Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,120 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Threading;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Console.OutputEncoding = System.Text.Encoding.Unicode;
+
+            MainMenu();
+
+        }
+
+        private static void MainMenu()
+        {
+            Console.Clear();
+            Console.WriteLine("Welcome! Choose one option");
+
+            var options = new List<string>
```

Comment:
Forget about using list in this chapter! 😄 For this specific case, array, in fact is better, because you have a predefined list of commands that you know. Since size won't change and all the commands are known- array is better.